docs/recovery.md

```diff
@@ -0,0 +1,351 @@
+# Table of Contents
+
+* [Recovering Funds From `lnd` (funds are safu!)](#recovering-funds-from-`lnd`-(funds-are-safu!))
+  * [On-Chain Recovery](#on-chain-recovery)
+    * [24-word Cipher Seeds](#24-word-cipher-seeds)
+    * [Wallet+Seed Passphrases](#wallet+seed-passphrases)
+    * [Starting On-Chain Recovery](#starting-on-chain-recovery)
+    * [Force In-Place Rescan](#force-in-place-rescan)
+  * [Off-Chain Recovery](#off-chain-recovery)
+    * [Obtaining SCBs](#obtaining-scbs)
+      * [On-Disk `channel.backup`](#on-disk-`channel.backup`)
+      * [Using the `ExportChanBackup` RPC](#using-the-`exportchanbackup`-rpc)
+      * [Streaming Updates via `SubscribeChannelBackups`.](#streaming-updates-via-`subscribechannelbackups`.)
+    * [Recovering Using SCBs](#recovering-using-scbs)
+
+# Recovering Funds From `lnd` (funds are safu!)
+
+In this document, we'll go over the various built-in mechanisms for recovering
+funds from `lnd` due to any sort of data loss, or malfunction. Coins in `lnd`
+can exist in one of two pools: on-chain or off-chain. On-chain funds are
+outputs under the control of `lnd` that can be spent immediately, and without
+any auxiliary data. Off-chain funds on the other hand exist within a 2-of-2
+multi-sig output typically referred to as a payment channel. Deepening on the
+exact nature of operation of a given `lnd` node, one of these pools of funds
+may be empty.
+
+Fund recovery for `lnd` will require two pieces of data: 
+  1. Your 24-word cipher seed
+  2. Your encrypted Static Channel Backup file (or the raw data)
+
+The SCB file is encrypted using a key _derived_ from the user's seed. As a
+result, it cannot be used in isolation.
+
+## On-Chain Recovery 
+
+### 24-word Cipher Seeds
+
+When a new `lnd` node is created, it's given a 24-word seed phrase, called an
+[`aezeed`](https://github.com/lightningnetwork/lnd/tree/master/aezeed). A valid
+see phrase obtained over the CLI `lncli create` command looks something like:
+```
+!!!YOU MUST WRITE DOWN THIS SEED TO BE ABLE TO RESTORE THE WALLET!!!
+
+---------------BEGIN LND CIPHER SEED---------------
+ 1. ability   2. noise   3. lift     4. document
+ 5. certain   6. month   7. shoot    8. perfect
+ 9. matrix   10. mango  11. excess  12. turkey
+13. river    14. pitch  15. fluid   16. rack
+17. drill    18. text   19. buddy   20. pool
+21. soul     22. fatal  23. ship    24. jelly
+---------------END LND CIPHER SEED-----------------
+
+!!!YOU MUST WRITE DOWN THIS SEED TO BE ABLE TO RESTORE THE WALLET!!!
+```
+
+### Wallet+Seed Passphrases
+
+During the creation process, users are first prompted to enter a **wallet
+password**:
+```
+Input wallet password:
+Confirm wallet password:
+```
+
+This password is used to _encrypt_ the wallet on disk, which includes any
+derived master private keys or public key data.
+
+Users can also _optionally_ enter a second passphrase which we call the _cipher
```

Comment:
above we refer to this as an "aezeed", here we call it a "cipher seed". imo we should stick to cipher seed throughout and link to the aezeed package above, what do you think?

Reply:
SGTM